pkg/processor/processor.go

```diff
@@ -178,6 +173,12 @@ func (runner *ReportRunner) Run(reportFn func(report *ReportToExecute) (map[stri
 	for _, report := range runner.Reports {
 		var err error
 
+		caseNumber, err := common.GetCaseNumberByFilename(report.File.Path)
+		if err != nil || caseNumber == "" {
+			logrus.Error(err)
```

Comment:
If the error is nil but the caseNumber is empty, the error returned will be 'nil' which will break whatever is calling this function into thinking that there are no errors.

Reply:
LGTM